hue_lo: 0
hue_hi: 70
sat_lo: 40
sat_hi: 255
val_lo: 140
val_hi: 255
trs_lo: 0
trs_hi: 255
